docs/plugins/plugin-zod/index.md

```diff
@@ -201,6 +201,11 @@ z.string()
 z.date()
 z.number()
 ```
+```typescript [{numbers: true}]
+z.string()
```

Comment:
z.string()
z.date()
is not needed here + can you also add example for date and string?

Reply:
My intent was to show that it's configurable individually.

Which would you prefer?

A block like the above for each data type, or a single block like:

```typescript [{numbers: true, strings: false, dates: false}]
z.string()
z.date()
z.coerce.number()
```

where dates and strings are explicitly false?

Reply:
Ooh that is also fine and less code in the docs. Your example here seems fine, thanks.